core/api/service/internal/internal_api.hpp

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_API_INTERNALAPI
+#define KAGOME_API_INTERNALAPI
+
+#include <boost/variant.hpp>
+
+#include "common/buffer.hpp"
+#include "outcome/outcome.hpp"
+#include "primitives/block_data.hpp"
+#include "primitives/block_header.hpp"
+#include "primitives/common.hpp"
+
+namespace kagome::api {
+
+  /**
+   * @class InternalApi provides doing various action over RPC
```

Comment:
Not a very useful comment to be honest. I'd suggest to say here that this is API specific to kagome, not part of polkadot JSON RPC API specification.